cmd/build.go

```diff
@@ -123,6 +134,13 @@ func newBuildCmd() *cobra.Command {
 		launch.ArtifactsDir,
 		"Path to the host side directory which is mounted into $SD_ARTIFACTS_DIR.")
 
+	buildCmd.Flags().StringVarP(
+		&memory,
+		"memory",
+		"m",
+		"",
+		"Memory limit for build container. Same as the option for docker.")
```

Comment:
Do we need to add the following message to the description which is written in design doc?
`Either b, k, m, g can be used as a size unit`

Reply:
It is more helpful, I will add the explain.